aconfigure.ac

```diff
@@ -2480,10 +2489,12 @@ AC_ARG_ENABLE(libwebrtc,
 			;;
 		     *win32* | *w32* | *darwin* | *linux*)
                          case $target in
-                             armv7l*gnueabihf)
-                                 ac_webrtc_instset=neon
-                                 ac_webrtc_cflags="-DWEBRTC_ARCH_ARM_V7 -mfloat-abi=hard -mfpu=neon"
-                                 ;;
+                             armv*gnueabihf)
+							 	if test "x$ax_cv_support_neon_ext" = "xyes"; then
```

Comment:
Also need to change the detection for webrtc_aec3 below?

Reply:
Not sure, though. With those minimal changes it compiles quite okay.

Reply:
webrtc_aec3 is not included to be built by default.
So test it with `--enable-libwebrtc-aec3` option.

Reply:
Implemented the same way as above. The change is really needed too, as long as --enable-libwebrtc-aec3 is present.